### DISCUSSION TITLE
Isar.dev still contains Documentation that refers to version 2.5.0.

### DISCUSSION THREAD

**Discussion by muddassar-lab** [isar/isar#680] [Help]
Isar.dev contains documentation that refers to stable version 2.5.0. When will it get updated to stable version 3.0.0?;

**Comment by simc** [comment]
It now has the latest docs

**Reply by colejd** [reply]
Hey @leisim, is there any way to view the 2.5.0 docs anymore? I have a web build going, and I'm using 2.5.0 since 3.0.0 isn't supported yet (#686).

**Reply by waffles9992** [reply]
you can try wayback machine!
https://web.archive.org/web/20220314105858/https://isar.dev/tutorials/quickstart.html
